src/BlazorWebView/src/Maui/Android/BlazorWebView.Android.cs

```diff
@@ -0,0 +1,24 @@
+using AWebView = Android.Webkit.WebView;
+using Android.Content;
+using Android.Views;
+using Android.Webkit;
+
+namespace Microsoft.AspNetCore.Components.WebView.Maui
+{
+	public class BlazorAndroidWebView : AWebView
+	{
+		public BlazorAndroidWebView(Context context) : base(context)
+		{
+		}
+
+		public override bool OnKeyDown(Keycode keyCode, KeyEvent? e)
+		{
+			if (keyCode == Keycode.Back && CanGoBack() && e?.RepeatCount == 0)
+			{
+				GoBack();
+				return true;
+			}
+			return false;
```

Comment:
We want to ensure the WebView captures the Android `back` button press before ceding control to the host app.